dev/codespaces-init.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+set -ex
+
+# install Nuke globally
+dotnet tool install Nuke.GlobalTool --global
```

Comment:
Can we use `dotnet tool restore` instead?

Reply:
OK

Reply:
https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/1720/commits/74d0e370c8d4c97978e2ae98b32422586756bd2c